Doc/Tutorial/chapter_uniprot.tex

```diff
@@ -25,15 +25,15 @@ \subsection{Parsing Swiss-Prot records}
 >>> from urllib.request import urlopen # Python 3 only
 >>> from io import TextIOWrapper
 >>> 
->>> handle = TextIOWrapper(urlopen("http://example.org/myprositefile.dat"))
+>>> handle = TextIOWrapper(urlopen("https://raw.githubusercontent.com/biopython/biopython/master/Tests/SwissProt/F2CXE6.txt"))
 \end{verbatim}
 to open the file stored on the Internet before calling \verb|read|.
 
 Note for Python 2 this can be done by using:
 \begin{verbatim}
 >>> from urllib import urlopen #Python 2 only
 >>>
->>> handle = urlopen("http://example.org/myprositefile.dat")
+>>> handle = urlopen("https://raw.githubusercontent.com/biopython/biopython/master/Tests/SwissProt/F2CXE6.txt")
```

Comment:
Never mind, reading the text it ought to have been a SwissProt file in the first place.